backend/src/main/java/com/tourneynizer/tourneynizer/controller/MatchController.java

```diff
@@ -37,4 +45,36 @@ public MatchController(MatchService matchService) {
             return new ResponseEntity<Object>(new ErrorMessage(e), new HttpHeaders(), HttpStatus.INTERNAL_SERVER_ERROR);
         }
     }
+
+    private static final Map<String, Object> toReturn = getToReturn();
+
+    private static Map<String, Object> getToReturn() {
```

Comment:
What is the purpose of this function and its corresponding object?

Reply:
He's gonna expose two different enum lists: one that is the actual enum names that we need to return and one that has user friendly enum strings. Porter is doing this because he doesn't want the front end to send back the user friendly strings, but instead the actual enum names.

Reply:
Gotcha.